google/cloud/forseti/services/inventory/base/cai_gcp_client.py

```diff
@@ -79,14 +197,16 @@ def iter_compute_disks(self, project_number):
         Yields:
             dict: Generator of Compute Disk.
         """
-        resources = self.dao.iter_cai_assets(
-            ContentTypes.resource,
-            'google.compute.Disk',
-            '//cloudresourcemanager.googleapis.com/projects/{}'.format(
-                project_number),
-            self.session)
+        cai_key_map = {
```

Comment:
How did you find all the key differences between CAI and GCP data?  I'm asking because I'm wondering how do we keep this up-to-date.

Reply:
This is a one time thing, it was gotten by comparing the public documentation with the internal proto representation for resources supported by CAI. It's not repeatable work, but should not need to be repeated. Specific details can be discussed outside of the pull request.